ckanext/blob_storage/tests/test_blob_storage_bug_fix.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for plugin.py."""
+import logging
+import mock
+import pytest
+
+# encoding: utf-8
+from ckan.plugins import toolkit
+from ckan.tests import factories, helpers
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.usefixtures('clean_db', 'with_plugins')
+class TestBlobStorageActivityDownload(object):
+    def test_can_download_release_resource_whether_it_exists_in_current_version_of_package_or_not(self, app):
+
+        user = factories.User()
+        dataset = helpers.call_action('package_create', name='dataset_for_bug_test')
+        resource = helpers.call_action('resource_create', package_id=dataset["id"])
+
+        # Now delete the resource
```

Comment:
you don't need to comment with what your code does. It should be clearly visible from the code itself. Like it does here.
Just remove it plz.